server/views/upload-document.html

```diff
@@ -46,31 +49,54 @@ <h2 id="helpTextSubHeading" class="govuk-heading-m">Upload file</h2>
             <li>smaller than {{ maximumFileSize }}MB</li>
           </ul>
 
-        {{ govukFileUpload({
-          attributes: {
-            accept: accept
-          },
-          id: "files",
-          name: "files",
-          errorMessage: fieldErrors['files']
-        }) }}
+          {{ govukFileUpload({
+            attributes: {
+              accept: accept
+            },
+            id: "files",
+            name: "files",
+            errorMessage: fieldErrors['files']
+          }) }}
+        </div>
+
+        <div id="uploading1" style="display:none">
+          <h2 id="uploadingHelpTextSubHeading" class="govuk-heading-m">Uploading file</h2>
+
+          <p id="uploadingHelpText" class="govuk-body">Your file is being processed<br>Please wait...</p>
+
+          <!-- style: large -->
```

Comment:
I don't think you need the comment here, you can tell it is "large" by the class name